PDEP-1: Purpose and guidelines for pandas enhancement proposals (#47444)
pandas-dev/pandas: web/pandas/pdeps/0001-purpose-and-guidelines.md

#### Accepted PDEP

A PDEP can only be accepted by the core development team, if the proposal is considered
worth implementing. Decisions will be made based on the process detailed in the
[pandas governance document](https://github.com/pandas-dev/pandas-governance/blob/master/governance.md).
In general, more than one approval will be needed before the PR is merged. And
there should not be any `Request changes` review at the time of merging.

Once a PDEP is accepted, any contributions can be made toward the implementation of the PDEP,
with an open-ended completion timeline. Development of pandas is difficult to understand and
forecast, being the contributors a mix of volunteers and developers paid from different sources,
with different priorities. For companies, institutions or individuals with interest in seeing a
PDEP being implemented, or to in general see progress to the pandas roadmap, please check how
you can help in the [contributing page](/contribute.html).

#### Implemented PDEP

Once a PDEP is implemented and available in the main branch of pandas, its
status will be changed to `Status: Implemented`, so there is visibility that the PDEP

> is not part of the roadmap and future plans, but a change that it already

Reviewer: ```suggestion
is not part of the roadmap and future plans, but a change that has already
```

happened. The first pandas version in which the PDEP implementation is

> available will also be included in the PDEP.

Reviewer: not yet relevant, but would this be just a note in the revision history or something else?

#### Rejected PDEP

A PDEP can be rejected when the final decision is that its implementation is

> not the best for the interests of the project. Rejected PDEPs are as useful as accepted

Reviewer: ```suggestion
not in the best interests of the project. Rejected PDEPs are as useful as accepted
```

PDEPs, since there are discussions that are worth having, and decisions about
changes to pandas being made. They will be merged with `Status: Rejected`, so
there is visibility on what was discussed and what was the outcome of the
discussion. A PDEP can be rejected for different reasons, for example good ideas
that aren't backward-compatible, and the breaking changes aren't considered worth
implementing.

#### Invalid PDEP

For submitted PDEPs that do not contain proper documentation, are out of scope, or
are not useful to the community for any other reason, the PR will be closed after

> discussion with the author, instead of merging them as rejected. This is to not
> add noise to the list of rejected PDEPs, which should contain documentation as

Reviewer: ```suggestion
discussion with the author, instead of merging them as rejected. This is to avoid
adding noise to the list of rejected PDEPs, which should contain documentation as
```
